### PR TITLE
NMA-1134 : fix the local currency

### PR DESCRIPTION
**Pull request by hadia**
## Issue being fixed or feature implemented
Fix the Local currency format to match figma 
image

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [x] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: wallet/src/de/schildbach/wallet/adapter/BuyAndSellDashServicesAdapter.kt [hunk 1]
```diff
@@ -40,16 +46,26 @@ class BuyAndSellDashServicesAdapter( val config: Configuration,val onClickListen
...
                 )
+
                 if(data.balance!=null){
                     binding.serviceBalance.setAmount(data.balance)
                 }
                 if(data.localBalance!=null){
-                    binding.serviceFiatAmount.setAmount(data.localBalance)
+                    binding.serviceFiatAmount.text = Constants.PREFIX_ALMOST_EQUAL_TO+  view.context.getString(
+                        R.string.fiat_balance_with_currency, fiatFormat.format(data.localBalance),
+                        currencySymbol
```
Comment (Syn-McJ) [inline wallet/src/de/schildbach/wallet/adapter/BuyAndSellDashServicesAdapter.kt:67]: Whether a currency symbol goes before or after an amount depends on locale of the device, but here it seems like it is always going to be after the amount.

**Review by Syn-McJ — CHANGES_REQUESTED** [review]

**Review by Syn-McJ — COMMENTED** [review]
See my comment

File: wallet/src/de/schildbach/wallet/adapter/BuyAndSellDashServicesAdapter.kt (hunk 1 above)
Comment (Syn-McJ) [reply wallet/src/de/schildbach/wallet/adapter/BuyAndSellDashServicesAdapter.kt:67]: Actually, I'm not sure if this is required given that we ignore this everywhere in the app except for the Enter Amount screens. @HashEngineering

Comment (hadia) [reply wallet/src/de/schildbach/wallet/adapter/BuyAndSellDashServicesAdapter.kt:67]: In the figma design it's before the amount

Comment (Syn-McJ) [reply wallet/src/de/schildbach/wallet/adapter/BuyAndSellDashServicesAdapter.kt:67]: Figma designs don't represent all the possible states of UI. Sometimes it has currency before the amount, sometimes after. Generally, it should depend on the device locale. NumberFormat class handles this automatically, but we probably need to use MonetaryFormat which doesn't.

**Review by Syn-McJ — APPROVED** [review]
Great work

**Review by HashEngineering — APPROVED** [review]

**Pull request merged**